docs/partner_editable/service_limits.adoc

```diff
@@ -4,14 +4,17 @@
 
 // Space needed to maintain table headers
 |VPCs |1
-|IAM roles |7
+|AWS Identity and Access Management (IAM) roles |7
 |Amazon S3 buckets |5
 |AWS Glue crawlers |4
 |AWS Glue jobs |3
 |Amazon Sagemaker notebook instances |1
 |Amazon Sagemaker endpoints |1
 |AWS Lambda functions |9
-|Amazon API gateway endpoints |1
+|Amazon API Gateway endpoints |1
 |Amazon Redshift instances |1
-|AWS Step functions |3
+|AWS Step Functions |3
+//TODO Shivansh, Should "Amazon Sagemaker notebook instances" be "Jupyter notebook instances" as in the archit. diagram?
+//TODO Shivansh, Should the archit. diagram show all five S3 buckets?
+//TODO SHivansh, For "AWS Step Functions" (the service name), what are the resources that we have three of?
```

Comment:
Step function itself is a resource type.

Reply:
@sshvans Sascha says in [this commit](https://github.com/aws-quickstart/quickstart-aws-utility-meter-data-analytics-platform/commit/167776badaf447c38471e787a561aac52a340caa) that the Step Functions resource we want to identify, which we have three of, is state machine.